src/modules/user.ts

```diff
@@ -32,23 +31,30 @@ export default class UserSource {
     });
 
     const { data } = await this.holaplex.query<
-      GetCustomerWalletData,
-      GetCustomerWalletVars
+      GetCustomerWalletsData,
+      GetCustomerWalletsVars
     >({
       fetchPolicy: 'network-only',
-      query: GetCustomerWallet,
+      query: GetCustomerWallets,
       variables: {
         project: process.env.HOLAPLEX_PROJECT_ID as string,
-        customer: user?.holaplexCustomerId as string,
-        assetType: process.env.HOLAPLEX_WALLET_ASSET_TYPE as AssetType
+        customer: user?.holaplexCustomerId as string
       }
     });
 
+    const assetTypes = process.env.HOLAPLEX_WALLET_ASSET_TYPE?.split(
+      ','
+    ) as AssetType[];
+
+    const wallets = data.project.customer?.treasury?.wallets?.filter((wallet) =>
+      assetTypes.includes(wallet.assetId as AssetType)
+    );
```

Comment:
Lets not worry about filtering out the other asset types for now just return the wallets form GetCustomerWallets.

Reply:
Also lets make wallets a field resolve on the user objects so its only queried from the API when requested by the client.